cloud/pkg/cloudhub/servers/factory.go

```diff
@@ -7,19 +7,23 @@ import (
 	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/channelq"
 	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/common/util"
 	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/servers/quicserver"
+	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/servers/udsserver"
 	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/servers/wsserver"
 )
 
 const (
 	ProtocolWebsocket = "websocket"
 	ProtocolQuic      = "quic"
+	ProtocolUDS       = "uds"
 )
 
 func StartCloudHub(protocol string, eventq *channelq.ChannelEventQueue, c *context.Context) {
 	if protocol == ProtocolWebsocket {
 		wsserver.StartCloudHub(util.HubConfig, eventq, c)
 	} else if protocol == ProtocolQuic {
 		quicserver.StartCloudHub(util.HubConfig, eventq, c)
+	} else if protocol == ProtocolUDS {
+		udsserver.StartServer(util.HubConfig, c)
```

Comment:
You may want to change the `if-else` items to swith case

Reply:
Done. Thanks